CHANGELOG.md

```diff
@@ -7,12 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-## v0.23.1 - 2025-09-15
+### enhancement
+- Update controller-runtime & k8s go client, address breaking changes @ramkrishankumarN @kondracek-nr [#581](https://github.com/newrelic/newrelic-infra-operator/pull/581)
 
-### dependency
-- Update golang.org/x/oauth2 to v0.27.0 @TmNguyen12 [#568](https://github.com/newrelic/newrelic-infra-operator/pull/568)
```

Comment:
Something wonky happened with the changelog & stuck this dependency entry, untransformed, under the v0.23.1 release. I just moved it under the transformed heading.